.github/workflows/data-builder-container.yml

```diff
@@ -47,9 +59,8 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v5
         with:
-
           push: ${{ github.event_name != 'pull_request' }}
           context: .
           file: ./docker/data.dockerfile
-          tags: ${{ github.event.inputs.docker_tag || steps.meta.outputs.tags }}
+          tags: ghcr.io/scilifelabdatacentre/swg-data-builder:${{ steps.set-tag.outputs.tag }}
```

Comment:
Neat! Is there a reason why the complete image name is now needed ?

Reply:
It's a bit weird that it seems to be needed but [when I tested on my own testing repo](https://github.com/RMCrean/multi-docker-action/commit/40ef254274076bfb678a5a4a57ac9d96c38574ea) (to prevent polluting this repo with failed builds/commits) it broke the build without it. 

I think this is because the docker/metadata-action@v5 we were using before handled this automatically, whereas with the custom bit of logic in Set Docker tag there is no automatic handling.